lib/cc/engine/analyzers/reporter.rb

```diff
@@ -0,0 +1,67 @@
+require 'cc/engine/analyzers/violation'
+
+module CC
+  module Engine
+    module Analyzers
+      class Reporter
+        def initialize(directory, language_strategy, io)
+          @directory = directory
+          @language_strategy = language_strategy
+          @io = io
+          @flay = Flay.new(flay_options)
```

Comment:
looking at this now, it seems like we use `mass_threshold` from the strategy - possibly more coupled to flay than I'd thought originally, it seems